packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1806,6 +1816,27 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     _cancelActivePointers();
   }
 
+  /// Method which sends the route name changes over System Channels.
+  ///
+  /// There are two methods: 'pushRoute', 'popRoute'. 'pushRoute' is also used
+  /// for notifying the route replacements.
+  ///
+  /// See also [SystemChannels.navigation], which handles subsequent navigation
+  /// requests.
+  void _sendRouteChangeToSystemChannels(String methodName, Route<dynamic> route, Route<dynamic> previousRoute) {
+    final String previousRouteName = previousRoute?.settings?.name;
+    final String routeName = route?.settings?.name;
+    if (previousRouteName != null || routeName!= null) {
```

Comment:
nit: add space between `routeName` and `!=`

Reply:
Done. Thanks!